confignode/src/main/java/org/apache/iotdb/confignode/persistence/AuthorInfo.java

```diff
@@ -56,6 +56,27 @@ public class AuthorInfo {
     }
   }
 
+  public TSStatus login(String username, String password) {
+    boolean status;
+    String loginMessage = null;
+    TSStatus tsStatus = new TSStatus();
+    try {
+      status = authorizer.login(username, password);
+    } catch (AuthException e) {
+      logger.info("meet error while logging in.", e);
+      status = false;
+      loginMessage = e.getMessage();
+    }
+    if (status) {
+      tsStatus.setCode(TSStatusCode.SUCCESS_STATUS.getStatusCode());
+      tsStatus.setMessage("Login successfully");
+    } else {
+      tsStatus.setMessage(loginMessage != null ? loginMessage : "Authentication failed.");
+      tsStatus.setCode(TSStatusCode.WRONG_LOGIN_PASSWORD_ERROR.getStatusCode());
+    }
```

Comment:
Is it possible that the user doesn't exist when AuthException occurs? You can add a TSStatusCode to express this~

Reply:
I will return WRONG_LOGIN_USERNAME_OR_PASSWORD_ERROR when the user does not exist or the password is wrong

Reply:
WRONG_LOGIN_PASSWORD_ERROR means that the username or password is wrong